glean-core/android/src/main/java/mozilla/telemetry/glean/scheduler/MetricsPingScheduler.kt

```diff
@@ -318,7 +318,7 @@ internal class MetricsPingScheduler(
  */
 internal class MetricsPingTimerTask(val scheduler: MetricsPingScheduler) : TimerTask() {
     companion object {
-        private const val LOG_TAG = "glean/MetricsPingTimerTask"
+        private const val LOG_TAG = "glean/MetricsPingTimer"
```

Comment:
Why change this and not the class name?

Reply:
This is to workaround a weird limitation: the log tag can only be 23 characters :(  The WorkManager-related class that used to be here had the same issue.

Reply:
Ah -- but now that I'm more caffeinated, I think I see what you're getting at -- let's change the class name as well so they match.